src/shared/ui/select/SelectList.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import {
+  DropdownContainer,
+  DropdownHeader,
+  DropdownListContainer,
+  DropdownListItem,
+  StyledIcon,
+} from '@/shared/ui/select/selectList.style';
+
+interface SelectListProps {
+  items: string[];
+  width: 'short' | 'medium' | 'long';
```

Comment:
이런 느낌으로 표현해 주는 건 어떨까요?
```suggestion
  width: "lg" | "md" | "sm" | "xs"
```